### ISSUE TITLE
Rate Limit Improvements

### ISSUE THREAD

**Issue by simlu** [dependabot/feedback#447]
I'm looking at the organization settings. I have set the pr limit per repository to one (see screenshot).

![rate_limit](https://user-images.githubusercontent.com/1539747/56604641-4efaf180-65b7-11e9-8b73-12812dfe9dd5.png)

When enabling live updates I would expect the next pr to be opened automatically directly after an existing one is merged (if there are further ones pending). However that is unfortunately not the case.

What would be needed to support that?

Cheers, L~

**Comment by greysteil** [comment]
This would be quite a bit of work on our side - Dependabot doesn't currently queue up updates for you, so it doesn't know what else it needs to update you on when you clear your queue. We could change that, but have some other big priorities to work on right now.

**Comment by simlu** [comment]
@greysteil I was afraid that would be the case. Might be more desirable to have batch updating then (as this is just a workaround for that use case). Do you have a high level roadmap you could share?

**Comment by greysteil** [comment]
Nothing super detailed. Doing minimal updates for security fixes has been top of our list, but is nearly finished now (just Composer to add it for). After than I want to think about verifying the version diffs Dependabot includes in its PRs (as detailed [here](https://dependabot.com/blog/stopping-dependencies-getting-hijacked-2/)), and then work on grouping updates.

A lot of time also goes into fixing any bugs we see - we try to keep Dependabot bug free because there's nothing worse than a dependency updating service that creates broken PRs!

**Comment by simlu** [comment]
@greysteil was there work done to improve this? My feeling is that something changed, but I'm not sure what

**Comment by greysteil** [comment]
I don't think so, but I'm not on the Dependabot team anymore (I work on security more generally at GitHub).
